src/services/postService.js

```diff
@@ -1,30 +1,56 @@
-const { throwIfError } = require('../utilities/dynamoUtilities');
+const { throwIfError, CLASS_POST } = require('../utilities/dynamoUtilities');
 const postDAO = require("../repository/postDAO");
 const uuid = require("uuid");
 
-
-async function createPost(userID, text, score, title){
-    const post = {class: "post", itemID: uuid.v4(), postedBy: userID, description: text, score, title, replies: [], likedBy: []};
+const createPost = async (userId, text, score, title) => {
+    const post = { class: CLASS_POST, itemID: uuid.v4(), postedBy: userId, description: text, score, title, replies: [], likedBy: [] };
     const data = await postDAO.sendPost(post);
     throwIfError(data);
     return post;
 }
 
-async function seePosts(){
+const createReply = async (userId, postId, text) => {
+    const post = await postDAO.getPost(postId);
+    if (!post.Item) {
+        throw { status: 400, message: `Post ${postId} doesn't exist` };
+    }
+    const reply = { itemID: uuid.v4(), postedBy: userId, description: text };
+    const data = await postDAO.sendReply(postId, reply);
+    throwIfError(data);
+    return reply;
+}
+
+const getPostById = async (postId) => {
```

Comment:
This is a duplicate function. There is a getPost(id) in dev already. When you merge you'll have to pick one and update all calls to use it